tests/unit/experimental/pipeline/test_kg_builder.py

```diff
@@ -248,60 +171,40 @@ async def test_knowledge_graph_builder_with_entities_and_file(_: Mock) -> None:
         from_pdf=True,
     )
 
-    internal_entities = [SchemaEntity(label=label) for label in entities]
-    internal_relations = [SchemaRelation(label=label) for label in relations]
-    assert kg_builder.entities == internal_entities
-    assert kg_builder.relations == internal_relations
-    assert kg_builder.potential_schema == potential_schema
+    # assert kg_builder.entities == entities
+    # assert kg_builder.relations == relations
+    # assert kg_builder.potential_schema == potential_schema
```

Comment:
Are these still true?

Reply:
No, the `kg_builder` does not need these attributes anymore. Will clean up these tests very soon!